python/py-game/Portfile

```
@@ -22,9 +22,10 @@ python.rootname pygame
 master_sites    pypi:p/pygame
 distname        pygame-${version}
 checksums       rmd160 abe4dc7b5e350617048474a3d168d9d1d09eae2b \
-                sha256 301c6428c0880ecd4a9e3951b80e539c33863b6ff356a443db1758de4f297957
+                sha256 301c6428c0880ecd4a9e3951b80e539c33863b6ff356a443db1758de4f297957 \
+                size 3223131
 
-python.versions 27 35 36 37
+python.versions 27 35 36 37 38
```

Any reason not to also add py39-game?
```suggestion
python.versions 27 35 36 37 38 39
```

---

It didn't compile for me with Python 3.9, problem is the obsolete "tp_print" being in the cython-generated C code already in the archive.
Maybe it could be regenerated with the macports cython, but I think v2 of pygame with a fix for this will be released pretty soon, so I decided not to try that.